<?xml version="1.0"?>
<launch>
  <arg name="camera_id" default="DEV_1AB22C00C28B" doc="The ID of the camera"/>
  <arg name="frame_id" default="pp_nir" doc="The frame id of the camera"/>
  <arg name="camera_info_url" default="file://$(find pp_alvium_driver)/calib/210722_nir.yaml"/>
  <arg name="exposure" default="2200" doc="Exposure (us)"/>
  <arg name="topic2publish" default="nir_raw" doc="Topic under which the images will be published"/>


<!--     <remap from="img_topic_subscribed" to="cam0/image_raw" /> -->
<!--     <remap from="pattern_topic_published" to="cam0/calib_pattern_recognised" /> -->
    <node pkg="pp_alvium_driver" name="pp_nir" type="mono_camera_node">
<!--         <rosparam file="$(find pp_alvium_driver)/config/params_rgb.yaml" command="load"/> -->
<!--         <remap from="pp/nir_raw" to="pp/rgb_raw"/> -->
        <param name="camera_id" value="$(arg camera_id)"/>
        <param name="camera_info_url" value="$(arg camera_info_url)"/>
        <param name="frame_id" value="$(arg frame_id)"/>
        <param name="exposure" value="$(arg exposure)"/>
        <param name="topic2publish" value="$(arg topic2publish)"/>
    </node>
</launch>
